frontend/src/hooks/reportMapper/cycleTime.ts

```diff
@@ -21,9 +21,9 @@ export const cycleTimeMapper = ({
     const swimlane = getSwimlaneByItemName(itemName)
     return swimlane
       ? [
-          { value: swimlane.averageTimeForSP, unit: Unit.PER_SP },
+          { value: swimlane.averageTimeForSP.toFixed(2), unit: Unit.PER_SP },
```

Comment:
extract `.foFixed(2)` as an util

Reply:
Sorry, I don't get the point why we need to wrap a basic function in another function, could you please give more details or example?

Reply:
Duplicated code